src/wroboclaw/util/mock.py

```diff
@@ -31,6 +31,26 @@ def _update_pos(self):
             self._current_pos = boundary_clamp(self.pos_clamp, self._current_pos, *self.pos_bounds)
         self._last_update_time = now
 
+    def _set_offset(self, offset: int):
+        """Sets an offset for the current encoder. Intended to be called before the motor starts
+        
+        Parameters
+        ----------
+        offset : int
+            The offset to be set for this encoder
+        """
+        self._current_pos = offset
+
+    def _set_counts_per_rotation(self, counts_per_rotation: int):
+        """Sets a value for the counts per rotation for the current encoder. Also intended to be called before the motor starts
+        
+        Parameters
+        ----------
+        counts_per_rotation: int
+            The counts per rotation for this encoder
+        """
+        self.pos_bounds = (0, counts_per_rotation)
+
```

Comment:
If these methods are supposed to be publicly accessible (and they are, you use them that way, that's ok), they should not use the 'private' method convention of starting with an underscore.

Reply:
Complete as of cb491a0.